br.com.eduk.metainfo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>br.com.eduk</id>
+  
+  <name>eduK</name>
+  <summary>A revolution in the social impact technology market, combining online education and income generation.</summary>
+  <url type="homepage">https://app.eduk.com.br</url>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  
+  <description>
+    <p>
+      Founded in 2013 and acquired by Mind Lab in 2021, eduK aims to address Brazil&apos;s dual challenge of future-focused education and rectifying past deficits through income generation. With the acquisition, Mind Lab set a goal to enable a million users to generate income by 2024, offering affordable access to over 3,200 courses and 9,000 hours of content from 1,100 experts to enhance skills and knowledge on any device. EduK has impacted over 5 million people and was recognized by HolonIQ in 2020 as one of Latin America&apos;s most innovative EdTech startups.
+    </p>
```

Comment:
I don't understand what this application does.

Reply:
eduK is a freemium SaaS educational platform, built with Electron, focused on online vocational courses. It offers over 3,200 courses, adopting a model that combines free access to selected content with subscription options for advanced features. Aimed at professional training, this is evidenced by the issuance of more than 100,000 certificates, and it promotes the professional success of its users by facilitating employment opportunities and business partnerships.